toolchain/check/eval.cpp

```diff
@@ -50,6 +50,11 @@ static auto GetPhase(SemIR::ConstantId constant_id) -> Phase {
   }
 }
 
+static auto GetTypePhase(Context& context, SemIR::TypeId type_id) -> Phase {
+  return context.types().GetConstantId(type_id).is_symbolic() ? Phase::Symbolic
+                                                              : Phase::Template;
```

Comment:
Should this CHECK that it's constant? (i.e., that the expectation of either Symbolic or Template is correct)

(also, might consider a brief comment on the function)

Reply:
Good call -- it turns out that this was also getting called when `type_id` is invalid (specifically for `StructTypeField` instructions that don't have a `type_id`). I've cleaned this up and added a check for that case.